nautobot/utilities/testing/__init__.py

```diff
@@ -114,6 +119,16 @@ def setUp(self):
         # Re-populate status choices after database truncation by TransactionTestCase
         populate_status_choices(apps, None)
 
+        # Create the test user and assign permissions
+        self.user = User.objects.create_user(username="testuser")
+        self.add_permissions(*self.user_permissions)
+
+        # Initialize the test client
+        self.client = Client()
+
+        # Force login explicitly with the first-available backend
+        self.client.force_login(self.user)
+
```

Comment:
Need to review why we're not just calling `super().setUpNautobot()` here - wasn't that the point of #1954?